index.bs

```diff
@@ -4485,11 +4486,11 @@ In the following, let \(N\) be the value of the
 
 	<pre nohighlight>
 	$$
-		X[k] = \frac{1}{N} \sum_{n = 0}^{N - 1} \hat{x}[n]\, e^{\frac{-2\pi i k n}{N}}
+		X[k] = \frac{1}{N} \sum_{n = 0}^{N - 1} \hat{x}[n]\, W^{-kn}_{N}
```

Comment:
Not really necessary, but in both Chrome and Firefox I find the original e^(-2 pi i k n / N} to be really, really small.  So use W to make the fonts come up a bit bigger and easier to read.